Example/FinancialConnections Example/FinancialConnectionsUITests/FinancialConnectionsUITests.swift

```diff
@@ -398,7 +398,7 @@ final class FinancialConnectionsUITests: XCTestCase {
 
         let usesLinkText = app.webViews
             .staticTexts
-            .containing(NSPredicate(format: "label CONTAINS 'uses Link to connect your account'"))
+            .containing(NSPredicate(format: "label CONTAINS 'Link'"))
```

Comment:
Can we use a different label on this screen that isn’t affected by the backend changes, or just generally wait for the page to be loaded?

Reply:
Good point - updated to look for the presence of `auth.stripe.com` in the top browser bar: 

<img width="688" alt="image" src="https://github.com/user-attachments/assets/23a2a497-e2de-4057-bf03-b2be0b0e0e09">